docs/conf.py

```diff
@@ -26,7 +26,6 @@
     "colon_fence",
 ]
 
-source_suffix = ".rst"
```

Comment:
Kristin and I discussed this line and felt that since we're using MyST as our primary language, this line is confusing and better omitted. this is equivalent to the default value anyway!